### PR TITLE
add fastify-graceful-shutdown plugin to readme

### PR DESCRIPTION
**Pull request by StarpTech**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,6 +101,7 @@ Bearer auth plugin for Fastify
 - [`fastify-cookie`](https://github.com/fastify/fastify-cookie) Parse and set cookie headers
 - [`fastify-env`](https://github.com/fastify/fastify-env) Load and check configuration
 - [`fastify-formbody`](https://github.com/fastify/fastify-formbody)
+- [`fastify-graceful-shutdown`](https://github.com/hemerajs/fastify-graceful-shutdown)
```
Comment (allevo) [inline README.md]: "Plugin to parse x-www-form-urlencoded bodies" may be wrong

**Comment by allevo** [comment]
@StarpTech  I don't understand: are you request my review on that plugin or for this PR?

**Comment by StarpTech** [comment]
Both if you ask like this ;)

File: README.md (hunk 1 above)
Comment (StarpTech) [reply README.md]: Oh... copy paste issue thanks

**Comment by StarpTech** [comment]
@allevo fixed

File: README.md [hunk 2]
```diff
@@ -101,7 +101,7 @@ Bearer auth plugin for Fastify
 - [`fastify-cookie`](https://github.com/fastify/fastify-cookie) Parse and set cookie headers
 - [`fastify-env`](https://github.com/fastify/fastify-env) Load and check configuration
 - [`fastify-formbody`](https://github.com/fastify/fastify-formbody)
-Plugin to parse x-www-form-urlencoded bodies
```
Comment (jsumners) [inline README.md]: You have removed the description for fastify-formbody.

**Comment by StarpTech** [comment]
@jsumners thanks it was very late yesterday :D

**Pull request merged**